pytorch_lightning/loggers/wandb.py

```diff
@@ -408,11 +408,12 @@ def log_text(
         self.log_table(key, columns, data, dataframe, step)
 
     @rank_zero_only
-    def log_images(self, key: str, images: List[Any], **kwargs: str) -> None:
+    def log_image(self, key: str, images: List[Any], **kwargs: str) -> None:
         """Log images (tensors, numpy arrays, PIL Images or file paths).
 
         Optional kwargs are lists passed to each image (ex: caption, masks, boxes).
         """
+        assert isinstance(images, list), f'Expected a list as "images", found {type(images)}'
```

Comment:
can you raise a TypeError instead? In an ideal world asserts only live in tests :)

Reply:
Done!